engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/repository-details-info.controller.js

```diff
@@ -36,6 +37,14 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
 
         $scope.progress = {uploading: false};
 
+        Setting.get({"search": "name = enable_deferred_download_policies"},
+        function (data) {
```

Comment:
Can you indent the function calls since they are parameters of the above Setting call?